docs/development_suite/set-up-infrastructure/configure-provider.mdx

```diff
@@ -206,13 +206,174 @@ A modal will appear allowing you to select one of the providers implementing tha
 
 Once you edit the default provider, you will be able to see the card with the new provider set.
 
-## Associate a Provider to a Project
+## Associate Providers to Projects
 
-If you haven't specified a default provider on a certain capability, or you wish to overwrite the default provider configuration, you will need to manually associate the provider with the projects you want to use it for within that company.
+If you haven't specified a default provider on a certain capability, or you wish to overwrite the default provider configurations on a specific project within the company, you will need to manually associate the provider with the project you want to use it for.
 
-To do this, you can use the CMS: navigate through the `Projects` section, select the desired project, and change the `providerId` field value in the `Repository` object.
+To do this, you can use the CMS: navigate through the `Projects` section, select the desired project, and edit the document by following the guidelines below, which specify what information to include depending on the provider type and capability.
 
-![Associate provider to project CMS](./img/associate-provider-to-project-cms.png)
+### Edit Git Provider
+
```

Comment:
Mettiamo un warning rossissimo in cui dobbiamo dire chiaramente che la migrazione dei dati non è gestita: se un utente vuol cambiare provider deve migrarsi a mano tutta la repo dal vecchio al nuovo provider!